examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -376,57 +405,73 @@ def _mean_frequency_by_risk_group(y_true, y_pred, sample_weight=None,
     axi.legend()
 plt.tight_layout()
 
-##############################################################################
-# The ``Ridge`` regression model can predict very low expected frequencies
-# that do not match the data. It can therefore severly under-estimate the risk
-# for some policyholders.
+###############################################################################
+# The dummy regression model predicts on constant frequency. This model is not
+# discriminative at all but is none-the-less well calibrated.
+#
+# The ``Ridge`` regression model can predict very low expected frequencies that
+# do not match the data. It can therefore severly under-estimate the risk for
+# some policyholders.
 #
-# ``PoissonRegressor`` and ``RandomForestRegressor`` show better consistency
-# between predicted and observed targets, especially for low predicted target
-# values.
+# ``PoissonRegressor`` and ``HistGradientBoostingRegressor`` show better
+# consistency between predicted and observed targets, especially for low
+# predicted target values.
 #
-# However, for some business applications, we are not necessarily interested
-# in the ability of the model to predict the expected frequency value, but
-# instead to predict which policyholder groups are the riskiest and which are
-# the safest. In this case, the model evaluation would cast the problem as a
-# ranking problem rather than a regression problem.
+# The sum of all predictions also confirms the calibration issue of the
+# ``Ridge`` model: it under-estimates by more than 3% the total number of
+# claims in the test set while the other three models can approximately recover
+# the total number of claims of the test portfolio.
 #
-# To compare the 3 models within this perspective, one can plot the fraction of
+# Evaluation of the discriminative power
```

Comment:
Mmh. Discrimination is a huge topic in insurance. Maybe we should use the word "ranking"?